main.js

```diff
@@ -0,0 +1,13 @@
+const list = document.querySelector("header nav ul li");
+list.remove();
+
+const span = document.querySelector(".featured span");
+span.remove();
+
+const article = document.querySelector("main .posts article");
+article.remove();
```

Comment:
Although you're passing the tests for this one, you're not actually removing the **last** article as the readme.md asks. Instead, you're removing the **first** article.
```suggestion
const allArticles = document.querySelectorAll("main .posts article");
let lastArticle = allArticles[allArticles.length -1]
lastArticle.remove()
```
`querySelector` grabs the **first** matching element, NOT the **last** matching element. In order to grab the **last** matching element out of several matching elements, you need to use `querySelectorAll` to get a node list of **all** matching elements. Node lists are very similar to arrays, so you can grab the last element of a node list in the same way that you'd access the last element of an array: `allArticles[allArticles.length -1]`.